salesagility/SuiteCRM
# Fixed #6340 - From Dropdown on Email Compose has utf-8 issue

## Description



![screenshot_1](https://user-images.githubusercontent.com/22373035/45881388-ae8be700-bdb3-11e8-8b08-da15bbe5c512.png)

## Motivation and Context

Issue reference: #6340 

## How To Test This

1. Open Email Compose.
2. Look at from dropdown.
3. Check that encoding is correct.

## Types of changes


### Final checklist
